docs/docs/contributing/testing.md

```diff
@@ -21,63 +17,39 @@ These tests can be found in `<sub-project>/test` and are used to check
 functionality of specific parts of the codebase in isolation e.g: parsing,
 scanning and message errors.
 
-Running a single unit test class from sbt is as simple as:
+To run all tests in e.g., for the compiler test-suite you can write:
 
 ```bash
-> testOnly absolute.path.to.TestClass
+> dotty-compiler/test
 ```
 
-You can further restrict the executed tests to a subset of `TestClass` methods
-as follows:
+To run a single test class you use `testOnly` and the fully qualified class name. 
```

Comment:
Actually, you can do:

```shell
> testOnly -- *targetFunction
```

to run all tests named `targetFunction`, in any class.

You can also do:

```shell
> testOnly *.MyTargetClass
```
to run all paths with a class named `MyTargetClass`. Both sides are regexes, so you should be able to do:

```shell
> testOnly * -- *
```

which would be the same as `test`.